spec/spec_helper.rb

```diff
@@ -2,7 +2,10 @@
 require 'simplecov-rcov'
 SimpleCov.formatter = SimpleCov::Formatter::RcovFormatter
 SimpleCov.minimum_coverage 100
-
+SimpleCov.start do
+  add_filter "/vendor/bundle/"
```

Comment:
@threedaymonk Turns out this isn't necessary at all -- simplecov seems to be ignoring `vendor/bundle` by default.  I hadn't appreciated this before owing to my habit of install gems in `.bundle`.  I'm just removing this altogether.